### ISSUE TITLE
Update for Atmos 19.0 please

### ISSUE THREAD

**Issue by FatalPunisher88** [Root-MtX/Nro2Nsp#56]
Update for Atmos 19.0 please.forwarders are broken and needs updating

**Comment by LyuboA31** [comment]
no need posting same issue twice i already opened one if he wants to fix it and have time he will we just have to wait until then just use AMS 0.18.1 and stay on 11.0.1 theres no need to update for now

**Comment by PHANTOMCLO** [comment]
not to be THAT guy, but sometimes switch users can accidentally hit the update button on the home menu before starting a software, or something. even i understand there is no need for updating, but i was kinda forced into it lol

**Comment by hearnia2k** [comment]
> not to be THAT guy, but sometimes switch users can accidentally hit the update button on the home menu before starting a software, or something. even i understand there is no need for updating, but i was kinda forced into it lol

You can just downgrade, or load your backup though; no problem.  So, this is a pretty weak point.  Plus if someone has 90DNS setup they won't be offered the update even, which if you're running CFW you probably should have something in place to prevent a ban, unless you don't care.  Rememebr, if you get banned you'll stop being offered updates too, as I understand, since you get banned from the CDN too now I think.

**Comment by cloavii** [comment]
fair enough. which ofw and atmosphere version should i downgrade to?

**Comment by chi-bat** [comment]
> > not to be THAT guy, but sometimes switch users can accidentally hit the update button on the home menu before starting a software, or something. even i understand there is no need for updating, but i was kinda forced into it lol
> 
> You can just downgrade, or load your backup though; no problem. So, this is a pretty weak point. Plus if someone has 90DNS setup they won't be offered the update even, which if you're running CFW you probably should have something in place to prevent a ban, unless you don't care. Rememebr, if you get banned you'll stop being offered updates too, as I understand, since you get banned from the CDN too now I think.

Not true, I've been CDN banned for a long time and I still get pestered about game and software updates.

**Comment by hearnia2k** [comment]
> > > not to be THAT guy, but sometimes switch users can accidentally hit the update button on the home menu before starting a software, or something. even i understand there is no need for updating, but i was kinda forced into it lol
> > 
> > 
> > You can just downgrade, or load your backup though; no problem. So, this is a pretty weak point. Plus if someone has 90DNS setup they won't be offered the update even, which if you're running CFW you probably should have something in place to prevent a ban, unless you don't care. Rememebr, if you get banned you'll stop being offered updates too, as I understand, since you get banned from the CDN too now I think.
> 
> Not true, I've been CDN banned for a long time and I still get pestered about game and software updates.

OK.  Perhaps they don't sue teh CDN for the meadata. However you won't be able to download the updates themselves since they are delivered via the CDN.  If you can't then it's not likely you're CDN banned, sicne you'd be accessing the CDN...

**Comment by chi-bat** [comment]
> > > > not to be THAT guy, but sometimes switch users can accidentally hit the update button on the home menu before starting a software, or something. even i understand there is no need for updating, but i was kinda forced into it lol
> > > 
> > > 
> > > You can just downgrade, or load your backup though; no problem. So, this is a pretty weak point. Plus if someone has 90DNS setup they won't be offered the update even, which if you're running CFW you probably should have something in place to prevent a ban, unless you don't care. Rememebr, if you get banned you'll stop being offered updates too, as I understand, since you get banned from the CDN too now I think.
> > 
> > 
> > Not true, I've been CDN banned for a long time and I still get pestered about game and software updates.
> 
> OK. Perhaps they don't sue teh CDN for the meadata. However you won't be able to download the updates themselves since they are delivered via the CDN. If you can't then it's not likely you're CDN banned, sicne you'd be accessing the CDN...

That's actually pretty odd, I'm CDN banned (can't access the eshop, youtube, any online game) but I was still able to update the console by total accident...

**Comment by hearnia2k** [comment]
> > > > > not to be THAT guy, but sometimes switch users can accidentally hit the update button on the home menu before starting a software, or something. even i understand there is no need for updating, but i was kinda forced into it lol
> > > > 
> > > > 
> > > > You can just downgrade, or load your backup though; no problem. So, this is a pretty weak point. Plus if someone has 90DNS setup they won't be offered the update even, which if you're running CFW you probably should have something in place to prevent a ban, unless you don't care. Rememebr, if you get banned you'll stop being offered updates too, as I understand, since you get banned from the CDN too now I think.
> > > 
> > > 
> > > Not true, I've been CDN banned for a long time and I still get pestered about game and software updates.
> > 
> > 
> > OK. Perhaps they don't sue teh CDN for the meadata. However you won't be able to download the updates themselves since they are delivered via the CDN. If you can't then it's not likely you're CDN banned, sicne you'd be accessing the CDN...
> 
> That's actually pretty odd, I'm CDN banned (can't access the eshop, youtube, any online game) but I was still able to update the console by total accident...

Youtube requires Nintendo authentication, not the Nintendo CDN.  eShop also requires authentication too.  So does online gaming.  I could be you simple have a console or account ban, and not a CDN ban.

**Comment by chi-bat** [comment]
> > > > > > not to be THAT guy, but sometimes switch users can accidentally hit the update button on the home menu before starting a software, or something. even i understand there is no need for updating, but i was kinda forced into it lol
> > > > > 
> > > > > 
> > > > > You can just downgrade, or load your backup though; no problem. So, this is a pretty weak point. Plus if someone has 90DNS setup they won't be offered the update even, which if you're running CFW you probably should have something in place to prevent a ban, unless you don't care. Rememebr, if you get banned you'll stop being offered updates too, as I understand, since you get banned from the CDN too now I think.
> > > > 
> > > > 
> > > > Not true, I've been CDN banned for a long time and I still get pestered about game and software updates.
> > > 
> > > 
> > > OK. Perhaps they don't sue teh CDN for the meadata. However you won't be able to download the updates themselves since they are delivered via the CDN. If you can't then it's not likely you're CDN banned, sicne you'd be accessing the CDN...
> > 
> > 
> > That's actually pretty odd, I'm CDN banned (can't access the eshop, youtube, any online game) but I was still able to update the console by total accident...
> 
> Youtube requires Nintendo authentication, not the Nintendo CDN. eShop also requires authentication too. So does online gaming. I could be you simple have a console or account ban, and not a CDN ban.

Apologies! I checked the gbatemp ban types chart and I'm indeed not CDN banned, I just have a normal ban. Once again I apologize for my ignorance.

**Comment by Kronos2308** [comment]
use https://github.com/StarDustCFW/SimpleNSPForwader for the momment 
is console based
